__tests__/index.test.js

```diff
@@ -169,8 +169,35 @@ describe("/api/articles tests", () => {
         expect(response.body.message).toBe("Invalid query parameter.");
       });
   });
+  test("200: accepts a topic query parameter.", () => {
```

Comment:
There is another test we could have. What about a topic that exists but has no articles? For example a new/unpopular topic. We should still get the array (of no articles) back.

This may mean you need an alternative way to validate your topic exists.